### ISSUE TITLE
OpenCV: TypeError: an integer is required (got type list)

### ISSUE THREAD

**Issue by m-alzam** [OlafenwaMoses/ImageAI#316]
Environment
=============
Python 3.7.4
ImageAI 2.1.3
opencv-python 4.1.0.25
tensorflow 1.13.1

Details
========
When running the following script using a custom model, ImageAI raises an error when drawing the rectangle of the recognized object.
```python
from imageai.Detection.Custom import CustomObjectDetection

detector = CustomObjectDetection()
detector.setModelTypeAsYOLOv3()
detector.setModelPath("chess-pieces-dataset/models/detection_model-ex-02--loss-25.59.h5") 
detector.setJsonPath("chess-pieces-dataset/json/detection_config.json")
detector.loadModel()


detections = detector.detectObjectsFromImage(input_image="test.jpg", output_image_path="test-detected.jpg")

for detection in detections:
    print(detection["name"], " : ", detection["percentage_probability"], " : ", detection["box_points"])
```
Error:
```python-traceback
File "[...]/venv/lib/python3.7/site-packages/imageai/Detection/Custom/__init__.py", line 1369, in draw_boxes_and_caption
    image_frame = cv2.rectangle(image_frame, (x1, y1), (x2, y2), class_color, 2)
TypeError: an integer is required (got type list)
```

**Comment by RyanThomas0** [comment]
Hi, did you end up solving this problem? I have the same issue currently

**Comment by m-alzam** [comment]
> Hi, did you end up solving this problem? I have the same issue currently

No, I don't. I drop the program since I couldn't manage to work.

**Comment by vrjmath** [comment]
Same issue

**Comment by kp67201** [comment]
same issue

**Comment by karthikkodakandla** [comment]
i am facing the same issue when using iamgeai : 

/usr/local/lib/python3.7/dist-packages/imageai/Detection/Custom/__init__.py:1234: RuntimeWarning: overflow encountered in exp
  return 1. / (1. + np.exp(-x))
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-55-9b093624b3ed> in <module>()
----> 1 detections = detector.detectObjectsFromImage(input_image='/content/drive/MyDrive/number_plate/train/images/Cars102.png', output_image_path='nplate1-detected.jpg')
      2 
      3 for obj in detections:
      4     print (obj['name'])
      5     print (obj['percentage_probability'])

1 frames
/usr/local/lib/python3.7/dist-packages/imageai/Detection/Custom/__init__.py in draw_boxes_and_caption(self, image_frame, v_boxes, v_labels, v_scores, show_names, show_percentage)
   1364             class_color = self.label_color(self.__labels.index(v_labels[i]))
   1365 
-> 1366             image_frame = cv2.rectangle(image_frame, (x1, y1), (x2, y2), class_color, 2)
   1367 
   1368             label = ""

TypeError: an integer is required (got type list)

**Comment by 2292269971** [comment]
Did you solve this issue?

**Comment by 2292269971** [comment]
> > 嗨，你最终解决了这个问题吗？我目前有同样的问题
> 
> 不，我没有。我放弃了该程序，因为我无法工作。

你解决了这个问题吗？
